cmd/rpcdaemon/README.md

```diff
@@ -263,6 +263,7 @@ The following table shows the current implementation status of Erigon's RPC daem
 | eth_submitHashrate                         | Yes     |                                      |
 | eth_getWork                                | Yes     |                                      |
 | eth_submitWork                             | Yes     |                                      |
+| eth_getVoteOnHash                          | Yes     | Relevant to bor consensus
```

Comment:
Looking at the original proposal: (https://forum.polygon.technology/t/proposal-improved-ux-with-milestones-for-polygon-pos/11534), it seems like this was intended to be **bor_getVoteOnHash**.  So we should clarify.

I think the other question about this api is if it is meant to be public as it seems like its intended to be called by heimdall only.

Reply:
Yes, I think it's mistakenly added in the `eth` namespace. It should be `bor_getVoteOnHash`. 

Yeah, so for now, we assume that validators only have their RPC exposed to their local heimdall. But yes, if the RPC is exposed mistakenly, it can be misused by external callers.